tools/perfscale-load-generator/watcher/watcher.go

```diff
@@ -122,6 +124,9 @@ func (w *ObjListWatcher) handleUpdate(obj interface{}) {
 		if vmi.Status.Phase == kvv1.Running {
 			w.addObj(string(vmi.GetUID()))
 		}
+		if vmi.Status.Phase == kvv1.Succeeded {
```

Comment:
What about `kvv1.Failed`? Shouldn't we also `w.deleteObj`?

Reply:
Yes, it should. Thanks!

Reply:
In fact, when the VM enters the "Failed" phase, it will be moved to the "Succeeded" phase later. Therefore, we must wait for the `Succeeded` phase to delete the obj entry.